### PR TITLE
Create Trainer models and APIs

### PR DESCRIPTION
**Pull request by gsvr**
https://trello.com/c/LKyhrSFQ/2-create-trainer-models-and-apis

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: emergingleaders_hub/urls.py [hunk 1]
```diff
@@ -6,6 +6,9 @@
     '',
     url(r'^grappelli/', include('grappelli.urls')),
     url(r'^admin/',  include(admin.site.urls)),
-    url(r'^operations/',
-        include('operations.urls')),
+    url(r'^api/v1/auth/',
+        include('rest_framework.urls', namespace='rest_framework')),
+    url(r'^api/v1/token-auth/',
+        'rest_framework.authtoken.views.obtain_auth_token'),
+    url(r'^api/v1/operations/', include('operations.urls')),
```
Comment (imsickofmaps) [inline emergingleaders_hub/urls.py]: drop `/operations` please

File: operations/tests.py [hunk 2]
```diff
@@ -43,17 +42,48 @@ def test_login(self):
...
 
-    def test_create_dummy_model_data(self):
+    def test_api_get_trainer(self):
+        trainer_data = {
+            "name": "Test User 1 Simple",
+            "msisdn": "+27820010001"
+        }
+        trainer = Trainer.objects.create(**trainer_data)
+        response = self.client.get(
+            '/api/v1/operations/trainers/%s/' % trainer.id,
```
Comment (imsickofmaps) [inline operations/tests.py]: Drop operations now

**Comment by imsickofmaps** [comment]
Minor then :+1:

**Pull request merged**